### PR TITLE
SCMOD-3427: Updated to build both a JDK and JRE base image

### PR DESCRIPTION
**Pull request by Anthony-McGreevy**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -22,13 +22,13 @@
...
-    <artifactId>opensuse-jre8-image</artifactId>
+    <artifactId>opensuse-java8-images</artifactId>
     <version>1.4.0-SNAPSHOT</version>
     <packaging>pom</packaging>
 
-    <name>openSUSE JRE8 image</name>
-    <description>An openSUSE-based image which includes the OpenJDK 8 Runtime Environment.</description>
-    <url>https://github.com/CAFapi/opensuse-jre8-image</url>
+    <name>openSUSE Java8 images</name>
+    <description>Two openSUSE-based images which either includes the OpenJDK 8 Runtime Environment or the OpenJDK 8 Development Kit.</description>
```
Comment (michael-bryson) [inline pom.xml]: "either includes" should be "either include"

File: src/main/docker/JRE8_Dockerfile [hunk 2]
```diff
@@ -1,6 +1,9 @@
 #
 # Copyright 2015-2017 EntIT Software LLC, a Micro Focus company.
 #
+#
+# Copyright 2015-2017 EntIT Software LLC, a Micro Focus company.
```
Comment (michael-bryson) [inline src/main/docker/JRE8_Dockerfile]: Duplicate license header?

**Review by michael-bryson — CHANGES_REQUESTED** [review]

**Review by dermot-hardy — CHANGES_REQUESTED** [review]
Rather than changing the filenames create a separate folder for each Dockerfile.

See https://github.com/docker-library/openjdk for an example.

File: pom.xml (hunk 1 above)
Comment (dermot-hardy) [reply pom.xml]: Actually just drop the "either" altogether.

File: pom.xml [hunk 3]
```diff
@@ -93,11 +93,31 @@
                 </executions>
                 <configuration>
                     <autoPull>always</autoPull>
+                    <verbose>true</verbose>
                     <images>
                         <image>
                             <name>${dockerCafImagePrefix}opensuse-jre8${dockerProjectVersion}</name>
                             <build>
-                                <dockerFileDir>.</dockerFileDir>
+                                <dockerFileDir>${project.basedir}/src/main/docker/JRE8</dockerFileDir>
```
Comment (dermot-hardy) [inline pom.xml:101]: Can't this just be the folder name - i.e. JRE8

**Review by dermot-hardy — COMMENTED** [review]
Can we change the folder names to just "jre" and "jdk" - lower-case, the way it is in the "openjdk" images I pointed at before.

File: src/main/docker/JRE8/Dockerfile [hunk 4]
```diff
@@ -18,13 +18,15 @@ FROM cafapi/opensuse-base:1
...
 RUN zypper -n refresh && \
-    zypper -n update && \
-    zypper -n install java-1_8_0-openjdk
+    zypper -n update
 
 # Install Java certificate installation script
 ADD https://raw.githubusercontent.com/CAFapi/caf-common/v1.14.0/container-cert-script/install-ca-cert-java.sh \
     /startup/startup.d/
 RUN chmod +x /startup/startup.d/install-ca-cert-java.sh
+
```
Comment (dermot-hardy) [inline src/main/docker/JRE8/Dockerfile:27]: Scrap the whitespace on this line.

**Pull request merged**